[2022-05-02 15:56:20,869] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: podsight_ETL.extract_podsight_data manual__2022-05-02T15:56:19.572835+00:00 [queued]>
[2022-05-02 15:56:20,879] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: podsight_ETL.extract_podsight_data manual__2022-05-02T15:56:19.572835+00:00 [queued]>
[2022-05-02 15:56:20,880] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 15:56:20,881] {taskinstance.py:1250} INFO - Starting attempt 1 of 6
[2022-05-02 15:56:20,881] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 15:56:20,894] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): extract_podsight_data> on 2022-05-02 15:56:19.572835+00:00
[2022-05-02 15:56:20,898] {standard_task_runner.py:52} INFO - Started process 309 to run task
[2022-05-02 15:56:20,901] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'podsight_ETL', 'extract_podsight_data', 'manual__2022-05-02T15:56:19.572835+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/podsight_ETL.py', '--cfg-path', '/tmp/tmpenezjn98', '--error-file', '/tmp/tmpgeiw85ql']
[2022-05-02 15:56:20,902] {standard_task_runner.py:80} INFO - Job 72: Subtask extract_podsight_data
[2022-05-02 15:56:20,952] {logging_mixin.py:109} INFO - Running <TaskInstance: podsight_ETL.extract_podsight_data manual__2022-05-02T15:56:19.572835+00:00 [running]> on host 53d127b53717
[2022-05-02 15:56:21,002] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=hamza.ahmed@liquidagency.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=podsight_ETL
AIRFLOW_CTX_TASK_ID=extract_podsight_data
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T15:56:19.572835+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-02T15:56:19.572835+00:00
[2022-05-02 15:56:21,004] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-02 15:56:21,005] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 "/Users/hamza.ahmed/Coding Projects/Python_Apps/Docker_Applications/Airflow_Docker_Apps/dags/Python_Scripts/podsight_API_JumpCloud.py"']
[2022-05-02 15:56:21,015] {subprocess.py:85} INFO - Output:
[2022-05-02 15:56:21,101] {subprocess.py:89} INFO - python3: can't open file '/Users/hamza.ahmed/Coding Projects/Python_Apps/Docker_Applications/Airflow_Docker_Apps/dags/Python_Scripts/podsight_API_JumpCloud.py': [Errno 2] No such file or directory
[2022-05-02 15:56:21,106] {subprocess.py:93} INFO - Command exited with return code 2
[2022-05-02 15:56:21,126] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 188, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2022-05-02 15:56:21,131] {taskinstance.py:1288} INFO - Marking task as UP_FOR_RETRY. dag_id=podsight_ETL, task_id=extract_podsight_data, execution_date=20220502T155619, start_date=20220502T155620, end_date=20220502T155621
[2022-05-02 15:56:21,145] {standard_task_runner.py:98} ERROR - Failed to execute job 72 for task extract_podsight_data (Bash command failed. The command returned a non-zero exit code 2.; 309)
[2022-05-02 15:56:21,158] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-05-02 15:56:21,193] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
